### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by vincentmaule** [OptimisticShaggy/LAB-101-html-more-kinds-of-content#1]
## Build a marketing page for an imaginary, soon-to-be-released game (you invent the details)

@egillespie Can you take a look at this? It's [hosted here](https://OptimisticShaggy.github.io/LAB-101-html-more-kinds-of-content/index.html) and meets the following criteria:

- [x] Include [a haiku](http://www.kidzone.ws/poetry/haiku.htm), describing the feelings this product will evoke (with `br` elements to add new lines where appropriate)
- [x] [Create and download a sound effect file](http://www.beepbox.co/) for what the game will sound like when you win, then include an audio player in the page so users can hear the sample
- [x] Include a table comparing which features your game has in comparison to competitors (these competitors may be real or made up)
- [x] Include a demo video from YouTube, but featuring a completely unrelated animal being cute (not getting hurt!)

**Comment by egillespie** [comment]
Wow, that background is intense! It looks like you're touching on all the requirements. The haiku, table, and video are all working as expected. 👍 

The audio element is there too, but the sound isn't playing. It looks like you are using a root-relative path to access the audio: `/audio/winning.wav`. When a browser sees a root-relative path (a path that starts with `/`), it will remove all of the routes from the current URL and put the path at the end.

In your case, the page URL is `https://optimisticshaggy.github.io/LAB-101-html-more-kinds-of-content/index.html` and the root-relative path `/audio/winning.wav` will try to be accessed at `https://optimisticshaggy.github.io/audio/winning.wav`.

Notice how the `/LAB-101-html-more-kinds-of-content` is removed from the URL when using the root-relative path? You can avoid this issue and help the browser find your file by removing the `/` at the beginning of `/audio/winning.wav`. Then the browser will take the full URL, host and routes, and add replace `index.html` with the relative path of `audio/winning.wav`:

```
https://optimisticshaggy.github.io/LAB-101-html-more-kinds-of-content/audio/winning.wav`
```

Which is the correct path to your audio file. 😎 

Would you mind updating the path to your audio file so the browser can find it and play the sound when I click the ⏯️  button?

Comment back when you're ready for me to take another look! 🔉

**Comment by vincentmaule** [comment]
Audio File has been updated! Root relatives are my go too because, it's helpful with understanding the directories and very simple for be. It's great to know that I need to use relative paths for Github Pages.

**Comment by egillespie** [comment]
Yeah, both types of paths shine in different situations. Your updated path looks and works great. Nice job! 🤘 :shipit:
